consensus/src/simplex/actors/voter/mod.rs

```diff
@@ -1465,6 +1460,32 @@ mod tests {
                     _ => panic!("unexpected batcher message"),
                 }
             }
+
+            // Now wait for our automaton to complete its proposal
+            // This should trigger `our_proposal` which will see the conflicting proposal
+            context.sleep(Duration::from_millis(100)).await;
+
+            // Add a notarization certificate for conflicting proposal
+            let (_, conflicting_notarization) =
+                build_notarization(&schemes, &namespace, &conflicting_proposal, quorum);
+            mailbox
+                .recovered(Certificate::Notarization(conflicting_notarization.clone()))
+                .await;
+
+            // Wait for a finalize vote to be broadcast (we drop our own conflicting proposal rather than marking as replaced)
```

Comment:
This is a little odd given we would otherwise refrain from voting (if we detected this on any other participant) 🤔

Reply:
(This is no change to existing logic...just making sure it is codified)